vars/ohpc-1.3

```diff
@@ -17,5 +17,5 @@ ohpc_slurm_packages:
     - "lmod-ohpc"
   database:
     - "slurm-slurmdbd-ohpc"
-
+openhpc_slurm_restart:
```

Comment:
I guess I am not sure why this is in vars vs defaults vs somewhere else. I must be getting confused.

Reply:
B/c it's only an internal flag we set, not a role variable.

Reply:
Hmm, I am not sure I have seen it done like this before, but I am no expert. I figured you just do it as a variable inside the play.

Should the default be false, or does that mess up overriding it? I guess None will evaulate to false throught he bool filter, but just curious.

Reply:
TBH I'll take advice on this. I thought I'd heard that defining everything in either vars or defaults was preferred, so that you could actually tell what variables the role used. But if not I can delete this, it is not functional.